core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -152,10 +152,10 @@
     </div>
   </div>
   <div class="oppia-content-wrapper oppia-recording-timer">
-    <div ng-if="isAudioAvailable && !isPlayingUploadedAudio()">
```

Comment:
Do we still need the `isPlayingUploadedAudio`? (in scope)

Reply:
Yes, we do for the other play and pause buttons in directive template.

Reply:
done.